book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,261 @@
+-   You do not have to agree to a 'terms of service' or 'end-user license agreement' with a third party to use and retain access to your lab book. These terms may be subject to unilateral alteration by that third party.
+
+If your provider of paper lab books goes out of business it has almost no bearing on your ability to continue doing your work.
+One paper notebook is much like another, finding a new provider is pretty easy.
+Changing providers does not impact on your ability to access your past notebooks or to continue operating with the same workflow in future ones.
+This is not necessarily true of ELNs.
+
+(rr-rdm-elns-choosing-archival)=
+### Archival function of notebooks
+
+Few active measures are needed to maintain the data in your paper notebooks. 
+The primary vulnerability of paper lab books is that there is only one copy. 
+Nevertheless, as long as they are kept in a cool, dry and dark spot (for example a fireproof safe) they will likely last decades.
+
+Electronic data requires much more active upkeep.
+Electronic data can, *if managed properly*, be more resilient to physical threats such as fire and flooding as it can frequently be backed up in multiple locations.
+A hard drive, or even a solid state drive, however, cannot be left in a draw for a decade or more and have a high likelihood of working without some amount of bit-rot or compatibility issues when plugged back in.
+
+Lab notebooks perform an archival function.
+When choosing an archival format one seeks to maximize the likelihood that one can recover the relevant information from that format.
+Proprietary formats are antithetical to this.
+Proprietary solutions generally obfuscate the format, even if just through absense of generally available specifications and documentation.
+Proprietary formats generally have a single implementation from their creator, this creates a single point of failure.
+Open formats can have multiple different implementations of tools which read them.
+If there is a good enough specification then new tools can even be written for the format if none of the originals are available anymore.
+Proprietary tools way require some kind of license activation process to use this may not be possible if their vendor no longer exists.
+Using a proprietary solution for archival purposes is therefore taking a needless risk with the future of your data.
+Your data's fate is tied to the format in which it is stored choose it wisely.
+
+(rr-rdm-elns-choosing-lockin)=
+### Lock-in
+
+Most proprietary or open tools will permit you to export your data in one format or another.
+The quality of this export is critical to scrutinise.
+The ability to save all your lab books as PDFs is fine but if it is the only option you may lose a lot of metadata, and the ability to import your data into a new solution. 
+The ability to export your data and retain the original structure is very important to evaluate.
```

Comment:
```suggestion
The ability to export your data and retain the original structure is very important.
```

Reply:
This is sometimes a matter of degree, you may be able to retain most of the information and the losses are acceptable to you when performing a migration. So the degree to which it is possible to 'losslessly' export your data is the criterion that I'm suggesting people consider when evaluating an ELN.